hedera/middleware.py

```diff
@@ -26,6 +30,19 @@ def __call__(self, request):
             if re.match(exemption, request.path):
                 response = self.get_response(request)
                 return response
+        if settings.IS_LTI and not request.user.is_authenticated:
+            try:
+                lti_user = login_existing_user(request)
+            except ObjectDoesNotExist:
+                request.session["lti_email"] = request.POST.get("lis_person_contact_email_primary", False)
```

Comment:
This seems fine to me. For semantics, though, I wonder where the lti param values can be made to default to `None` instead of `False`. For example, `request.session["lti_email"] = request.POST.get("lis_person_contact_email_primary", None)`. Defaulting to `False` might imply that a value of type `bool` is expected. Defaulting to `None` implies that something is expected but none was found.

Reply:
Good point